app/src/main/res/layout/item_voice_commands.xml

```diff
@@ -13,6 +13,6 @@
         tools:text="Open WhatsApp"
         android:id="@+id/voiceCommand"
         android:padding="@dimen/padding_moderate"
-        android:textSize="@dimen/text_size_large"/>
+        android:textSize="@dimen/text_size_medium"/>
```

Comment:
Is just changing the textsize doing the work?

Reply:
Yes it is!